app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { withTheme } from 'styled-components'
+import { th } from '@pubsweet/ui'
+import Select from 'react-select'
+
+const Root = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: ${th('gridUnit')};
+`
+
+const SelectLimitMessage = styled.p`
+  color: ${th('colorSuccess')};
+`
+
+const eLifeOptions = [
+  {
+    value: 'biochemistry-chemical-biology',
+    label: 'Biochemistry and Chemical Biology',
+  },
+  { value: 'cancer-biology', label: 'Cancer Biology' },
+  { value: 'cell-biology', label: 'Cell Biology' },
+  {
+    value: 'chromosomes-gene-expression',
+    label: 'Chromosomes and Gene Expression',
+  },
+  {
+    value: 'computational-systems-biology',
+    label: 'Computational and Systems Biology',
+  },
+  {
+    value: 'developmental-biology-stem-cells',
+    label: 'Developmental Biology and Stem Cells',
+  },
+  { value: 'ecology', label: 'Ecology' },
+  {
+    value: 'epidemiology-global-health',
+    label: 'Epidemiology and Global Health',
+  },
+  { value: 'evolutionary-biology', label: 'Evolutionary Biology' },
+  { value: 'genetics-genomics', label: 'Genetics and Genomics' },
+  {
+    value: 'human-biology-medicine',
+    label: 'Human Biology and Medicine',
+  },
+  {
+    value: 'immunology-inflammation',
+    label: 'Immunology and Inflammation',
+  },
+  {
+    value: 'microbiology-infectious-disease',
+    label: 'Microbiology and Infectious Disease',
+  },
+  { value: 'neuroscience', label: 'Neuroscience' },
+  { value: 'physics-living-systems', label: 'Physics of Living Systems' },
+  { value: 'plant-biology', label: 'Plant Biology' },
+  {
+    value: 'structural-biology-molecular-biophysics',
+    label: 'Structural Biology and Molecular Biophysics',
+  },
+]
+
+const findOptionByValue = (value, options) =>
+  options.find(option => option.value === value)
+
+export class ThemelessSubjectAreaDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    const savedOptions = this.props.savedValues.map(value =>
+      findOptionByValue(value, eLifeOptions),
+    )
+
+    this.state = {
+      selectedOptions: savedOptions,
+      hasReachedMultiselectLimit: false,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
i think these are already bound since you're using what they call property initializers (arrow funcs as class methods)

Reply:
Ah yeah. Didn't clock that we had `transform-class-properties` in our babel setup :+1: 

Most of our components are stateless, so this doesn't really come up. But we do have a few stateful ones. One of these [does contain this binding pattern](https://github.com/elifesciences/elife-xpub/blob/develop/app/components/submission/ComponentWithSaving.js#L15), which is partly why I went with it - for consistency.

Also, as far as I understand, `transform-class-properties` means that you don't need to use a constructor - see https://www.fullstackreact.com/articles/use-property-initializers-for-cleaner-react-components/ but we have this in most of our stateful components.

Is it perhaps worth deciding on a common convention that our team is going to use for these things?